jenkins/scripts/files/run_fetch_logs.sh

```diff
@@ -60,4 +68,14 @@ then
   sudo chown -R ${USER}:${USER} "${LOGS_DIR}/upgrade"
 fi
 
+if [[ "${TESTS_FOR}" == "feature_tests" || "${TESTS_FOR}" == "feature_tests_centos" ]]
+then
+  target_config=$(sudo find /tmp/ -type f -name "kubeconfig*")
+  if [ "${target_config}" != "" ]
+  then
+    #fetch target cluster k8s logs
+    fetch_k8s_logs "target" $target_config
```

Comment:
I would use more readable directory naming
'target_cluster' instead of 'target'

Reply:
done